rest/call/list-get-example-6/list-get-example-6.5.x.cs

```diff
@@ -0,0 +1,25 @@
+// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Types;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        TwilioClient.Init(accountSid, authToken);
+
+        var status = CallResource.StatusEnum.Busy;
+        var to = new PhoneNumber("+15558675309");
+        var calls = CallResource.Read(status: status, to: to);
```

Comment:
In this example you don't need the variables `status` and `to`, the context is provided by the named parameters. Would you mind removing them please?